packages/cli/src/runners/task-runner-process.ts

```diff
@@ -147,11 +127,7 @@ export class TaskRunnerProcess extends TypedEmitter<TaskRunnerProcessEventMap> {
 	async forceRestart() {
 		if (!this.process) return;
 
-		if (this.useLauncher) {
-			await this.killLauncher(); // @TODO: Implement SIGKILL in launcher
-		} else {
-			this.process.kill('SIGKILL');
-		}
+		this.process.kill('SIGKILL');
```

Comment:
It needs to be the orchestrator that does it. n8n has no visibility or access to it. With the launcher being there in-between, it would have to provide a health endpoint that probes the runner's health endpoint. I created https://linear.app/n8n/issue/CAT-345/implement-healthcheck-for-the-go-launcher for this

Reply:
Lovely, thank you